e2e/api/steps.js

```diff
@@ -383,7 +382,7 @@ const assertError = async (pageId, eventData, expectedErrorMessage, responseBody
   assert.equal(response.status, 422);
   assert.equal(responseBody.message, responseBodyMessage);
   if(responseBody.callbackErrors != null){
-    assert.equal(responseBody.callbackErrors[0], expectedErrorMessage);
+    assert.include(responseBody.callbackErrors[0], expectedErrorMessage);
```

Comment:
I think we should leave it as `equal` here, as we would rather assert specific error message.